src/AdafruitIO.cpp

```diff
@@ -165,10 +165,21 @@ const __FlashStringHelper* AdafruitIO::statusText()
   }
 }
 
-void AdafruitIO::run(uint16_t busywait_ms)
+aio_status_t AdafruitIO::run(uint16_t busywait_ms)
 {
+  uint32_t timeStart = millis();
+  aio_status_t net_status = networkStatus();
+  // If we aren't connected, return network status -- fail quickly
+  // Previous version would just hang on a network error
```

Comment:
Thanks for explaining, but get rid of L173. Don't want to ship with this. It'll go into release notes.